tests/acceptance/26_cf-net/serial/cf-net_connect_2.cf

```diff
@@ -0,0 +1,60 @@
+body contain myshell
+{
+    !windows::
+        useshell => "useshell";
+    windows::
+        useshell => "powershell";
+}
+
+body common control
+{
+    inputs => {
+        "../../default.cf.sub",
+        "../../run_with_server.cf.sub"
+    };
+    bundlesequence => { default("$(this.promise_filename)") };
+}
+
+bundle agent init
+{
+    methods:
+        "any" usebundle => dcs_fini("$(G.testdir)/cf-net_connect.log");
+
+        "any" usebundle => generate_key;
+        "any" usebundle => trust_key;
+
+        "any" usebundle => start_server("$(this.promise_dirname)/serverd_admit_localhost_15000.srv.cf");
+    files:
+        "$(G.testdir)/policy_server.dat"
+            create => "true",
+            edit_line => insert_lines("127.0.0.1:15000");
+
+}
+
+bundle agent test
+{
+    commands:
+        "$(sys.cf_net)"
+            args => " --workdir '$(G.testdir)' --verbose connect  > $(G.testdir)/cf-net_connect.log  2>&1",
+            contain => myshell;
+}
+
+bundle agent check
+{
+    vars:
+        "lines" int => countlinesmatching(".*","$(G.testdir)/cf-net_connect.log");
+    classes:
+        "expected_output_seen" expression => regline("Connected & authenticated successfully to '127.0.0.1:15000'", "$(G.testdir)/cf-net_connect.log");
+        "more_than_one_line" expression => isgreaterthan("$(lines)", 1);
+    methods:
+        expected_output_seen & !more_than_one_line ::
```

Comment:
@jimis Here's the docs, RTFM :) https://docs.cfengine.com/docs/3.10/reference-language-concepts-classes.html#operators-and-precedence